src/lib/configs.ts

```diff
@@ -20,7 +22,9 @@ export async function initializeConfigs(config: Config = {}): Promise<{ [key: st
   GLEE_DIR = path.resolve(GLEE_PROJECT_DIR, '.glee')
   GLEE_LIFECYCLE_DIR = path.resolve(GLEE_DIR, config.functionsDir || 'lifecycle')
   GLEE_FUNCTIONS_DIR = path.resolve(GLEE_DIR, config.functionsDir || 'functions')
-  GLEE_CONFIG_FILE_PATH = path.resolve(GLEE_DIR, 'glee.config.js')
+  GLEE_CONFIG_FILE_PATH_TS = path.resolve(GLEE_DIR, 'glee.config.ts')
+  GLEE_CONFIG_FILE_PATH_JS = path.resolve(GLEE_DIR, 'glee.config.js')
+  GLEE_CONFIG_FILE_PATH = existsSync(GLEE_CONFIG_FILE_PATH_TS) ? GLEE_CONFIG_FILE_PATH_TS : GLEE_CONFIG_FILE_PATH_JS
```

Comment:
In case they both exist, it would be awesome to log a message saying that `glee.config.js` exists but it's not being used in favor of `glee.config.ts` and the user might want to migrate the JS file or remove it.

Have a look at this [blog post](https://wix-ux.com/when-life-gives-you-lemons-write-better-error-messages-46c5223e1a2f) before writing the message.

To log the message in a nice way, you can use the [logWarningMessage](https://github.com/asyncapi/glee/blob/master/src/lib/logger.ts#L98) function that Glee already provides.

Let me know if you need help. I'm here or on [Slack](https://asyncapi.com/slack-invite).

Reply:
Just pushed an update. Let me know if it works 😄